documents/motions/21.2021.01 - WCA Documents.md

```diff
@@ -0,0 +1,27 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 21.2021.1  
+**Subject:** WCA Documents  
+**Intent:** Establish priorities and amendment procedures of WCA Documents  
+**Submitted by:** Motion has been removed and all responsibilities will be absorbed by the WCA Results Team.  
+**Date:** February 28, 2021  
+
+# Motion
+
+1. In case of any dispute between two WCA Documents the following order of priority shall be established:
+  1. WCA Bylaws
+  2. WCA Motions
+  3. WCA Regulations and Guidelines
+  4. WCA Policies
+  5. Other WCA material
+2. In case of any dispute between two WCA Documents with the same priority, the WCA Board shall determine which document takes precedence.
+3. In case of any dispute between two translations of the same document the American English version shall prevail
+4. Amendments of WCA Documents:
+  1. WCA Bylaws shall be amended by the procedures set out in section 12 of the Bylaws.
+  2. WCA Motions shall be amended by the procedures set out in Motion 13.2020.1 Amendments of Motions.
```

Comment:
This should be Motion 13.2021.1 instead.

Reply:
Thanks, resolved.